freqtrade/exchange/exchange.py

```diff
@@ -181,24 +196,34 @@ def set_sandbox(self, api, exchange_config: dict, name: str):
                                      "Please check your config.json")
                 raise OperationalException(f'Exchange {name} does not provide a sandbox api')
 
-    def _load_async_markets(self) -> None:
+    def _load_async_markets(self, reload=False) -> None:
         try:
             if self._api_async:
-                asyncio.get_event_loop().run_until_complete(self._api_async.load_markets())
+                asyncio.get_event_loop().run_until_complete(
+                    self._api_async.load_markets(reload=reload))
 
         except ccxt.BaseError as e:
             logger.warning('Could not load async markets. Reason: %s', e)
             return
 
-    def _load_markets(self) -> Dict[str, Any]:
+    def _load_markets(self, reload=False) -> None:
         """ Initialize markets both sync and async """
         try:
-            markets = self._api.load_markets()
-            self._load_async_markets()
-            return markets
+            self._api.load_markets(reload=reload)
+            self._load_async_markets(reload=reload)
```

Comment:
I don't think we should reload async-markets too, but leave that to CCXT if it thinks it's needed.

We only consume the sync-version at the moment, the async-version is just so ccxt works correctly for async requests.

foring the refresh for both sync and async will cause double-requests to the exchange which should return the same result.